src/plan/gencopy/gc_work.rs

```diff
@@ -209,29 +212,3 @@ impl<VM: VMBinding> DerefMut for GenCopyMatureProcessEdges<VM> {
         &mut self.base
     }
 }
-
-#[derive(Default)]
```

Comment:
Was this code unused?

Reply:
The code is unused at the very beginning (when GenCopy is merged for the first time). GenCopy uses `scheduler::gc_work::ProcessModBuf` instead.